stdlib/public/Darwin/Foundation/String.swift

```diff
@@ -96,3 +96,23 @@ extension Substring : _ObjectiveCBridgeable {
 }
 
 extension String: CVarArg {}
+
+/*
+ This is on NSObject so that the stdlib can call it in StringBridge.swift
+ without having to synthesize a receiver (e.g. lookup a class or allocate)
+ */
+private extension NSObject {
+  @_effects(releasenone)
+  @objc(newTaggedNSStringWithASCIIBytes_:length_:)
+  func createTaggedString(bytes: UnsafePointer<UInt8>,
+                          count: Int) -> AnyObject? {
+    //TODO: update this to use _CFStringCreateTaggedPointerString once we can
+    return CFStringCreateWithBytes(
+      kCFAllocatorSystemDefault,
+      bytes,
+      count,
+      CFStringBuiltInEncodings.UTF8.rawValue,
+      false
+    ) as NSString as NSString?
```

Comment:
Casting directly to AnyObject here causes swiftc to emit a bridging call to bridge the optional out to objc; using an objc type explicitly avoids that

Reply:
I'd suggest translating these PR comments to source code comments to lessen the pain of future generations. This all seems like valuable info.